src/app/commentators/commentators.html

```diff
@@ -0,0 +1,31 @@
+<div class="container">
+    <div class="matchup">
+        <div>
+            <span *ngIf="vm.player1_seed">({{ vm.player1_seed }})</span> {{ vm.player1 }}
+			vs
+			<span *ngIf="vm.player2_seed">({{ vm.player2_seed }})</span> {{ vm.player2 }}
+        </div>
+    </div>
+	<div class="matchType">
+        <div>
+		{{ vm.matchType }}
+		</div>
+    </div>
+	<div class="commentary1">
+        <div>
+            Spikevegeta
+        </div>
+    </div>
+	<div class="commentary2">
+        <div>
+            JHobz296
+        </div>
+    </div>
+    <div>
+        <div class="logo">
+            <div>
+                <img src="../assets/SRUlogo.png"/>
+            </div>
+        </div>
+    </div>
+</div>
```

Comment:
I just wanna make sure I'm understanding this right: CommentatorsComponent is for the scene with the commentator webcams, right? If so, it might need to be general enough with layout to fit either 1 or 2 webcams.